### PR TITLE
GOVERNMENT.md: Mission Statements and Bylaws

### PR DESCRIPTION
**Pull request by sstucker**
I want to collect feedback on this draft following our exciting discussion at fNIRS 2022

### CODE REVIEW

File: GOVERNMENT.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+The mission of the SNIRF government is to ensure the format's longevity and to encourage its adoption. Critical to this mission is the ability of the fNIRS community to expand or correct the specification as new developments emerge without creating issues and conflicts for users. The SNIRF government is responsible for moderating this process, and for collecting and disseminating important information to SNIRF users.
+
+## Bylaws
+
+## Article I. Steering Committee
+
+**Section 1.** Steering Committee Definition. The SNIRF Steering Committee is ultimately responsible for guiding the development of SNIRF in accordance with its mission. The committee must unanimously approve the releases of official and final changes to the specification, as well as changes to the SNIRF government or mission statement.
+
+**Section 2.** Election. Steering committee members must be nominated or self-nominated in advance of the end of a sitting member's term. The departing member is responsible for the organization of the election, and cannot be elected to remain in the same seat. All SNIRF Contributors, Maintainers, and Steering Committee members are elligble to vote in the election.
```
Comment (dboas) [inline GOVERNMENT.md:20]: check spelling of 'eligible'

File: GOVERNMENT.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+**Section 4.** Meetings. A majority of Steering Committee members must meet at least biannually. At least one Maintainer must be present at these meetings. These meetings should review the following:
+- The most recent Maintainers Report
+- Any pending releases or critical changes
+- project priorities and progress towards goals
+- Delegation of resources (i.e maintainers' attention) towards completion of goals
+
+The meeting should result in the authorship of a brief report.
+
+**Section 5.** Steering Committee Reports. The Steering Committee must author a brief report corresponding following each of its required biannaual meetings. This report must document the actions and considerations of the committee, and must be archived and disseminated via the GitHub repository. The report must include the names of the Steering Committee's members.
```
Comment (dboas) [inline GOVERNMENT.md:32]: delete "corresponding"

File: GOVERNMENT.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+**Section 6.** GitHub Repository Role. Steering Committee members must have the role of `Admin` on the GitHub repository.  
+
+**Section 7.** Specification Releases. The Steering Committee must unanimously approve of official releases and publish them via GitHub. Releases must be thoroughly documented and properly formatted as described in RELEASE.md. The Committee is subsequently responsible for alerting the community to the release.
+
+**Section 8.** Maintainer Election. Any member of the Steering Committee may at any time approve the election of a Maintainer. The Steering Committee member must give the Maintainer the role of `Maintain`.
```
Comment (dboas) [inline GOVERNMENT.md:38]: what do you mean by "approve the election of a Maintainer."? One the one hand it sounds like any steering committee member can add a maintainer at any time. But on the other hand, what you say is they can approve an election of a maintainer. I think you mean the former, but maybe you mean the latter. If you mean the latter, I guess this means a steering committee member at anytime to start an election with one nominee that goes out to the email list of contributors to then vote for that one nominee to become a maintainer. Is that what you mean?

File: GOVERNMENT.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+**Section 3.** Terms. Maintainers serve 2 year terms. At the end of their term, they must step down or seek the renewal of their title from a member of the Steering Committee.
+
+**Section 4.** Maintainers' Report. Maintainers must collectively author a biannual report summarizing the prominent actions and considerations of the Maintainers since the previous report. This report must include any critical issues or pending releases which the Steering Committee must consider. 
+
+**Section 5.** GitHub Repository Role. Maintainers must have the role of `Maintainer` on the GitHub repository.
+
+## Article III. Additional Bylaws
+
+**Section 1.** Pull Requests. Maintainers are ultimately responsible for the merging of pull requests into the working branches of the SNIRF specification, but both Steering Committee Members and Maintainers have the authority to do so. A Steering Committee members and Maintainers may NOT merge their own pull requests, but should instead request the review of another contributor.
```
Comment (dboas) [inline GOVERNMENT.md:54]: replace "review of another" with "review by another"

File: GOVERNMENT.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+**Section 2.** Scheduling of Meetings. Meetings may be scheduled and organized by anyone. No fewer than 4 months and no more than 7 months should elapse between meetings such that they remain biannual.
```
Comment (dboas) [inline GOVERNMENT.md:56]: I would remove "No fewer than 4 months and". Why prevent meetings from happening more frequently if they are needed?

Comment (rob-luke) [reply GOVERNMENT.md:56]: Agreed

**Comment by rob-luke** [comment]
> Do you want to place lower limits on:
> 
> * the time for which PRs remain open, in order to ensure that all have time to comment?
> * the number of approvals before a PR can be merged, e.g. two. (this can be enforced easily on GH)?

Great ideas. I suggest two weeks for PR to remain open. And two approvals for a PR.

File: GOVERNMENT.md (hunk 3 above)
Comment (sstucker) [reply GOVERNMENT.md:38]: I meant that they can onboard one at any time. I will revise for clarity

**Comment by rob-luke** [comment]
I just reviewed this again and it looks good @dboas @samuelpowell @Horschig.

**Review by samuelpowell — APPROVED** [review]

**Review by Horschig — APPROVED** [review]

File: GOVERNMENT.md (hunk 2 above)
Comment (dboas) [inline GOVERNMENT.md]: I think we have to delete "corresponding" from the first sentence so that it reads "...brief report following each of ..."

**Review by dboas — APPROVED** [review]

**Pull request merged**